src/sessionDurationsToMinutes.ts

```diff
@@ -0,0 +1,16 @@
+function hoursToMinutes(hours: number): number {
+    return Math.round(hours * 60)
+}
+
+export const sessionDurationsToMinutes = (
+    { session_lifetime, idle_session_lifetime }: { session_lifetime?: number, idle_session_lifetime?: number }): { session_lifetime_in_minutes?: number, idle_session_lifetime_in_minutes?: number } => {
+    const sessionDurations: {
+        session_lifetime_in_minutes?: number
+        idle_session_lifetime_in_minutes?: number
+    } = {};
+
+    if (!!session_lifetime) sessionDurations.session_lifetime_in_minutes = hoursToMinutes(session_lifetime)
```

Comment:
This conditional is what prevents this null bug from occurring. If either of the session durations are undefined, they won't be returned.